EnemyCollision.cs

```diff
@@ -39,4 +40,12 @@ void Shoot ()
     }
 
     // TODO: Problem 3: Define a public class named Weapon. In the class, define 3 int variables for arrow, sword, and rocket.
+    public class Weapons
+    {
+        int arrow; //can also be int arrow = 1; if value needs to be set now
```

Comment:
These are also defined correctly. Some programmers might like to define the variables with public/private in front. But that is up to you.